src/main/java/seedu/duke/util/Parser.java

```diff
@@ -237,6 +237,14 @@ private Command prepareDeleteNote(String userMessage) {
         }
     }
 
+    private Command prepareFind(String userMessage) throws SystemException {
```

Comment:
Is it possible to catch the exception with this function and return a new IncorrectCommand("Missing info" or "No keyword") here

Reply:
Addressed 👍